src/pages/Gift/components/Category.tsx

```diff
@@ -24,7 +34,7 @@ const Category = () => {
       <Title>선물 테마</Title>
       <List>
         {themes.data?.map((category) => (
-          <Item key={category.themeId}>
+          <Item key={category.themeId} onClick={() => goThemes(category.themeId)}>
```

Comment:
[제안] Item을 Link 요소로 생성해서 Link의 `to` API를 사용해도 좋을 것 같네요
```suggestion
          <Item key={category.themeId} to={...}>
```